.travis.yml

```diff
@@ -0,0 +1,40 @@
+dist: trusty
+
+language: go
+
+go:
+  - 1.7
+  - tip
+
+go_import_path: github.com/docker/containerd
+
+addons:
+  apt:
+    packages:
+      - btrfs-tools
+
+install:
+  - wget https://github.com/google/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip -O /tmp/protoc-3.1.0-linux-x86_64.zip
+  - unzip -o -d /tmp/protobuf /tmp/protoc-3.1.0-linux-x86_64.zip
+  - export PATH=$PATH:/tmp/protobuf/bin/
+  - go get golang.org/x/net/context
```

Comment:
`go get` will be no longer needed when https://github.com/docker/containerd/pull/390 gets merged